spec/Entity/CreditMemoSpec.php

```diff
@@ -58,4 +59,9 @@ function it_has_units(): void
     {
         $this->getUnits()->shouldBeLike([new CreditMemoUnit('Portal gun', 1000, 50)]);
     }
+
+    function it_has_date_of_creation(): void
+    {
+        $this->getIssuedAt()->shouldHaveType(\DateTime::class);
```

Comment:
Maybe we can pass `new \DateTime('01-01-2020 10:10:10')` at `let method` and check `$this->getIssuedAt()->shouldBeLike(new \DateTime('01-01-2020 10:10:10'))` here?